roles/alexandria-v2/templates/environment.rb.j2

```diff
@@ -0,0 +1,7 @@
+# Load the Rails application.
+require File.expand_path('../application', __FILE__)
+
+Rails.env = ENV['RAILS_ENV'] || '{{ rails_env }}'
```

Comment:
Okay, sounds like a good topic for a team conversation. The three changes (add the ansible template, replace environment.rb with environment.rb.template, and add environment.rb as a shared file for capistrano) should happen together if we adopt that approach. @jcoyne @val99erie @mark-dce @ilessing thoughts?